CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## [unreleased]
+
+### Fixed
+
+* If a module is marked as private `.ts` file will not be generated.
```

Comment:
Please follow the convention of the previous change log.
> - Was it described in terms of what was modified rather than how it was modified?
> - Did it start with the past tense? ex) Fixed, Updated